### DISCUSSION TITLE
Is there support for the Arduino GIGA?

### DISCUSSION THREAD

**Discussion by fletchjp** [TcMenu/TaskManagerIO#55] [Q&A] [answered]
STM32H747XI

https://thepihut.com/products/arduino-giga-r1-wifi

Thank you

**Comment by davetcc** [comment, marked as answer]
It is an mbed based core, so yes, I would imagine that it _should_ work out of the box, with the caveat I don't have one to test it on.

**Reply by fletchjp** [reply]
Thank you. I am tempted to order one. By the way, I do run TaskmanagerIO on Arduino DUEs. John

**Reply by fletchjp** [reply]
Hi. I now have an Arduino GIGA. I am running a simple example of TaskManagerIO. The GIGA defines ARDUINO_ARCH_MBED and also ARDUINO_ARCH_MBED_GIGA. Experiments continue! John
